cirq/sim/density_matrix_simulator.py

```diff
@@ -618,10 +619,10 @@ def __init__(self,
     def _value_equality_values_(self):
         measurements = {k: v.tolist() for k, v in
                         sorted(self.measurements.items())}
-        return (self.params, measurements, self.final_simulator_state)
+        return (self.params, measurements, self._final_simulator_state)
 
     def __repr__(self):
         return ("cirq.DensityMatrixTrialResult(params={!r}, measurements={!r}, "
-                "final_simulator_state={!r})"
-                .format(self.params, self.measurements,
-                        self.final_simulator_state))
+                "final_simulator_state={!r})".format(
```

Comment:
If we don't want to expose the final simulator state, why are we including it in the trial results objects and printing it out?

Reply:
This is a repr. We want it to generate the code needed to create the object, no?  If it were __str__ I'd agree.